src/QsCompiler/LanguageServer/global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "3.1.100",
+    "rollForward": "latestFeature"
+  }
+}
```

Comment:
We should have only one global.json that applies to the compiler repository. Currently, the global.json at the root of the repository also sets the SDK version to 3.1, but it may be updated to support 5 soon. In that case, running `dotnet` from the LanguageServer folder will require 3.1, but may use 5 when run from any other folder.

Since this global.json is intended to only be used by the language server after it's built and running, is it possible for it to have a different name in the source folder that `dotnet` won't recognize, but have it be renamed to global.json in the output directory?